src/screens/Profiles/ProfileScreen.js

```diff
@@ -163,7 +173,7 @@ class ProfileScreen extends Component {
     return item;
   }
 
-  renderInfoBlock = () => {
+  renderInfoBlock() {
```

Comment:
This style with the `= () => {` doesn't harm performance and I tend to use it throughout my components. In `render` methods it's not actually necessary, but in event handlers it is or your `this` is bound incorrectly so I tend to just use it everywhere. It still only creates the function once on instantiation of the component, so in general I'd leave this in place unless I'm missing something.

Reply:
I agree but indeed most of these are not my work, so I was just formatting them to have a decent common code style. I'll change these to arrow versions 👍